packages/workflow-core/src/lib/plugins/external-plugin/vendor-consts.ts

```diff
@@ -754,12 +754,12 @@ export const BALLERINE_API_PLUGIN_FACTORY = {
             transformer: 'jmespath',
             mapping: `{
               ${options.dataMapping || ''}
-              kybCompanyName: entity.data.additionalInfo.companyName,
-              customerCompanyName: entity.data.additionalInfo.customerCompany,
+              kybCompanyName: entity.data.additionalInfo.companyName || entity.data.companyName,
+              customerCompanyName: entity.data.additionalInfo.customerCompany || entity.data.customerCompany,
```

Comment:
Tech debt until the mapping of customerCompany and companyName is fixed on the new entity component.